src/UI/Root.re

```diff
@@ -105,6 +119,22 @@ let make = (~state: State.t, ()) => {
       <KeyDisplayerView state />
     </Overlay>
     statusBar
+    {switch (contextMenu) {
+     | Some(model) =>
+       let onOverlayClick = () =>
+         GlobalContext.current().dispatch(ContextMenuOverlayClicked);
+       let onItemSelect = item =>
+         GlobalContext.current().dispatch(ContextMenuItemSelected(item));
+
+       <ContextMenu.Overlay
+         theme
+         font=uiFont
+         model
+         onOverlayClick
+         onItemSelect
+       />;
```

Comment:
This is what renders the actual menu, above a clickable overlay that dismisses the menu.

Reply:
Cool! Looks great